README.md

```diff
@@ -1 +1,25 @@
-# squirrel_prediction
+# squirrel_prediction repository
+
+
+Repository for prediction and learning related SQUIRREL packages.
+
+Installation requiremets
+```
+sudo apt-get install scipy
```

Comment:
and python-scipy ?? https://github.com/ros/rosdistro/blob/1389b578f8005ea3bc914da29629be63cb0a4234/rosdep/python.yaml#L1909